development-defaults.ini

```diff
@@ -165,9 +165,7 @@ stripe_error = "Stripe Error Override"
 
 [[door_payment_method]]
 cash   = "Pay with cash"
-stripe = "Pay with credit card now (faster)"
-manual = "Pay with credit card at the registration desk (slower)"
-group  = "Taking an unassigned Group badge (group leader must be present)"
+manual = "Pay with credit card at the registration desk"
```

Comment:
Actually never mind, I thought that ``c.DOOR_PAYMENT_METHOD_OPTS`` was used in ``models.py`` for the ``payment_method`` column, but on closer examination that's actually using ``c.PAYMENT_METHOD_OPTS`` which is not affected by this change.

My point (which is not actually valid for this PR) is that we need to leave that unchanged or else we'd get weird errors at confusing times.